cegs_portal/search/templates/search/v1/experiment_list.html

```diff
@@ -117,4 +225,5 @@ <h1>
     downloadDataSetup("{{ csrf_token }}");
 </script>
 {% endif %}
+<script src="https://cdn.jsdelivr.net/npm/tw-elements/js/tw-elements.umd.min.js"></script>
```

Comment:
t we already load this in a particular way elsewhere: `<script src="{% static 'js/tw-elements/index.min.js' %}"></script>`

Reply:
Right. Thought there were issues with it before but turns out another error was causing the problem. Need to add this back.